src/main/resources/synthetic.xml

```diff
@@ -16,21 +16,34 @@
     <type type="wp.War" extends="was.War" description="JSR Portal WAR archive">
         <property name="portalAppName" label="Portal Application Name" description="Name for this portal application" category="Portal"/>
         <property name="portalAppUid" label="Portal Application Identifier"
-                  description="Unique identifier for this portal application as defined in the portal.xml" category="Portal"/>
+                  description="Unique identifier for this portal application as defined in the portlet.xml" category="Portal"/>
         <property name="portalAppConcreteUid" required="false" label="Concrete Portal Application Identifier"
-                  description="Required when deploying a IBM-API type portlet. Concrete Unique identifier for this portal application as defined in the portal.xml"
+                  description="Required when deploying a IBM-API type portlet. Concrete Unique identifier for this portal application as defined in the portlet.xml"
                   category="Portal"/>
         <property name="portlets" kind="set_of_ci" required="true" as-containment="true" referenced-type="wp.PortletSpec" label="Portlets" category="Portal"
                   description="List of portlets to register."/>
     </type>
+    <type type="wp.PortletSpec" extends="udm.BaseEmbeddedDeployable" description="Portlet spec">
+        <property name="portletName" description="Name of the portlet (portlet-name) as described in the portlet.xml"/>
+        <property name="uniqueName" description="Unique name for portlet" required="false"/>
+        <property name="preferences" description="Portlet preferences." required="false" kind="map_string_string"/>
+        <property name="clones" kind="set_of_ci" required="true" as-containment="true" referenced-type="wp.PortletCloneSpec" label="Clones" description="List of portlet clones to register."/>
```

Comment:
This property should be made optional, not required.  Otherwise existing deployments that don't use clones will fail.